src/hw_isolation_record/manager.cpp

```diff
@@ -185,7 +185,28 @@ std::optional<sdbusplus::message::object_path> Manager::createEntry(
                           _bus, entryObjPath, *this, recordId, severity,
                           resolved, associationDeftoHw, entityPath)));
 
-        utils::setEnabledProperty(_bus, isolatedHardware, resolved);
+        // TODO:https://github.com/ibm-openbmc/openpower-hw-isolation/issues/39
```

Comment:
**Commit message:**

> The enable property will remain false for now until the isolation
servier restarts and records are refreshed.

typo "service"?

> It is difficult to reproduce the issue, so tested by micking throwing
exception before D-Bus call to PLDM to enable the property.

typo "mimicking"?

Reply:
ok in commit message i thought in the code fixed now